libpod/runtime.go

```diff
@@ -475,13 +475,32 @@ func NewRuntimeFromConfig(configPath string, options ...RuntimeOption) (runtime
 	// Set three fields not in the TOML config
 	runtime.config.StateType = defaultRuntimeConfig.StateType
 	runtime.config.OCIRuntime = defaultRuntimeConfig.OCIRuntime
-	runtime.config.StorageConfig = storage.StoreOptions{}
+
+	storageConf, err := util.GetDefaultStoreOptions()
+	if err != nil {
+		return nil, errors.Wrapf(err, "error retrieving rootless storage config")
```

Comment:
would it always be rootless at this point?

Reply:
Yes this should just say storage config.

Reply:
fixed